notebooks/2.1.1-he-rain-analysis.ipynb

```diff
@@ -72,22 +72,31 @@
    "metadata": {},
```

Comment:
<p>should be "the below line is <strong>the percent of circles </strong>who had at least one station reporting but those stations could not agree"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataKind-DC/audubon-cbc/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='100'/>

Reply:
<p>Fixed in Rain, snow, and snowfall notebooks</p>